reporter.go

```diff
@@ -233,6 +235,12 @@ func (r *remoteReporter) Close() {
 // reporting new spans.
 func (r *remoteReporter) processQueue() {
 	timer := time.NewTicker(r.bufferFlushInterval)
+	close := func() {
```

Comment:
Not a huge fan of assigning to built-in function name `close`. Also, can't you just do `defer func() {...}()` in this case?

Reply:
+1, `defer func() {...}()` should do it